lib/rails_semantic_logger/extensions/sidekiq/sidekiq.rb

```diff
@@ -1,70 +1 @@
-# Sidekiq patches
-if Sidekiq::VERSION.to_i == 4
```

Comment:
Ok this whole file of patches is _really_ old it turns out lol or its patching really old Sidekiq versions is more accurate lol

We are on Sidekiq 7.* so just removing this from our fork to avoid it being really confusing